bin/configure-tower-projects.py

```diff
@@ -606,15 +606,19 @@ def generate_compute_environment(self, name: str, model: str) -> dict:
                         "maxCpus": 1000,
                         "gpuEnabled": False,
                         "ebsAutoScale": True,
-                        "allowBuckets": [],
+                        "allowBuckets": [
```

Comment:
What is this for?

Reply:
This list determines which buckets will be mounted on the AWS Batch EC2 instances using Fusion.